Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml

```diff
@@ -0,0 +1,4 @@
+name: RedHatEventsCollectorBrokerVM
+id: RedHatEventsCollectorBrokerVM
+fromversion: 3.3.0
```

Comment:
We used  6.8.0 in all the XSIAM integrations we did

Reply:
https://github.com/demisto/content/pull/19252/commits/e0b068ed0f1e7c2b3cea5e2ce94c1487713c1ad1